src/pytorch_lightning/trainer/connectors/checkpoint_connector.py

```diff
@@ -161,9 +161,10 @@ def _set_ckpt_path(
             ckpt_path = getattr(self.trainer.checkpoint_callback, "best_model_path", None)
 
         if ckpt_path == "last":
-            candidates = [getattr(ft, "ckpt_path", None) for ft in ft_checkpoints] + [
-                getattr(cb, "last_model_path", None) for cb in self.trainer.checkpoint_callbacks
-            ]
+            candidates = [getattr(ft, "ckpt_path", None) for ft in ft_checkpoints]
+            for callback in self.trainer.checkpoint_callbacks:
+                if hasattr(callback, "_find_last_checkpoints"):
```

Comment:
For customization, we'd want to have this as something the users can subclass and override with some guarantees. I guess this is why you chose to use `hasattr`.

But I would check `isinstance(ModelCheckpoint)` instead in this PR

Reply:
Yeah, that makes sense since we probably cannot expect people to implement this in the same way as us if they subclass a checkpoint. Done